src/styles/styles.module.scss

```
@@ -0,0 +1,36 @@
+@import './CircularStd/fonts';
+@import './Roboto/fonts';
+@import './EudoxusSans/fonts';
+@import './Avenir/fonts';
+
+.page {
+  font-family: 'Avenir';

```

здесь и где ещё встретишь: указывай, пожалуйста, fallback-шрифт, чтобы он загрузился, если с загрузкой основного произошли проблемы:
```suggestion
  font-family: 'Avenir', sans-serif;

```